src/command_modules/azure-cli-botservice/azure/cli/command_modules/botservice/custom.py

```diff
@@ -31,7 +31,7 @@
 def create(cmd, client, resource_group_name, resource_name, kind, description=None, display_name=None,  # pylint: disable=too-many-locals
            endpoint=None, msa_app_id=None, password=None, tags=None, storageAccountName=None,
            location='Central US', sku_name='F0', appInsightsLocation='South Central US',
-           language='Csharp', version='v3'):
```

Comment:
Is this a breaking change?  Generally, changing defaults is considered as such. Additionally, if you change the default to None will the service fail or will it choose one?  If it chooses one, you may wish to do that instead so you don't need to update your CLI code whenever the service default changes.

Reply:
@stevengum 	Per the issue, users *must* specify `-v v4` if they want a v4 bot. Wouldn't it make more sense to default to v4 unless the user specifies `-v v3` (which this still allows, I believe...unless there's an `msbot` command it breaks, maybe)? Regardless, if this is being addressed later, should we just close this PR and the issue?

Reply:
@mdrichardson If I create a script that calls `az bot create -n TestBot -k webapp --location westus` it creates a v3 bot. I now expect that I get a v3 bot when I call my script.

If I update Azure CLI and take a version of the CLI with this PR's changes, my old script now creates a v4 bot. This is a breaking change because my previous script no longer works the way I expect it to.

Similarly for `az bot publish`, the `--version` arg changes the way we prepare to publish a bot. Changing the default would break users.

I agree with the original issue, it _should_ default to v4. However we shouldn't take it at this time and this PR should be closed. The issue should be retained to outwardly show that we're aware of it, and I've updated the issue to show that it's in need of a sprint.